### PR TITLE
Fix fileupload prompt

### PR DESCRIPTION
**Pull request by MattBudz**
**Spec**
Currently when copy + pasting an image into dradis it will request to rename the image multiple times based on the number of views visited. This is caused by turbolinks re-binding the event on each page load.

**Solution**
Unbind the event on each page unload

**Extra**
I also noticed that large uploaded images overflow the containing div when displayed within a note, evidence, etc. This is fixed here as well.

> I assign all rights, including copyright, to any future Dradis
> work by myself to Security Roots.

### Check List

- [x] Added a CHANGELOG entry

### CODE REVIEW

File: app/assets/javascripts/tylium/behaviors.js.coffee [hunk 1]
```diff
@@ -216,3 +219,8 @@ document.addEventListener "turbolinks:load", ->
   $('[data-behavior~=smooth-scroll]').on 'click', ->
     target = $(this).data('target');
     $('[data-id~=' + target + ']')[0].scrollIntoView({behavior: "smooth"});
+
+# Un-bind fileUpload on page unload.
+document.addEventListener "turbolinks:before-cache", ->
```
Comment (aapomm) [inline app/assets/javascripts/tylium/behaviors.js.coffee]:
```suggestion
document.addEventListener 'turbolinks:before-cache', ->
```

File: app/views/layouts/tylium.html.erb [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <% if flash.any? %>
       <% # when flash messages present, disable turbolinks cache to avoid
         # the message appear and disappear in the next page load %>
-      <meta name="turbolinks-cache-control" content="no-cache">
+      <meta name="turbolinks-cache-control" content="reload">
```
Comment (etdsoft) [inline app/views/layouts/tylium.html.erb:14]: why is this needed?

File: app/assets/javascripts/tylium/behaviors.js.coffee [hunk 3]
```diff
@@ -217,3 +220,9 @@ document.addEventListener "turbolinks:load", ->
     history.pushState({}, '', this.href);
     $(this.hash)[0].scrollIntoView behavior: 'smooth'
     return
+
+# Un-bind fileUpload on page unload.
+document.addEventListener 'turbolinks:before-cache', ->
```
Comment (etdsoft) [inline app/assets/javascripts/tylium/behaviors.js.coffee:225]: I think this code needs to be moved closer to the initialiser block:

- It saves the `if ().length` check
- It makes it easier to understand
- It allows us to potentially extract the entire behaviour elsewhere.
